src/website/app/demos/Tooltip/examples/prose.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Tooltip from '../../../../../Tooltip';
+
+export default {
+  id: 'prose',
+  title: 'In Prose',
+  description: `Tooltips can add information to prose.`,
+  scope: { Tooltip },
+  // TODO: this should probably be in a <p> tag, but that causes errors
```

Comment:
The error is because `div` is not a valid HTML descendant of `p`. Unfortunately, these all render as `div`s: Popover*, PopoverTrigger, PopoverContent*, CardBlock.

The ones with a * are styled react-popper components, which are supposed to accept a `component` prop to render as something other than the default 'div'. I tried setting it to 'span' and it didn't work, though. The others I was able to change to `span`s just fine.

I think this is a bigger issue than just the demo, as it's likely a Tooltip will appear inside a `p` in a consuming app.